roachtest: quiescence with dead node

tbg opened the pull request:
This test currently fails, at least in remote mode (curiously in local
mode it works, at least on my laptop).

```
--- FAIL: kv/quiescence/nodes=3 [unstable] (1258.49s)
        kv.go:160: QPS dropped by more than 80%: from 2669.32 to 1582.14
```

Fixing this test is the goal for #9446.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/26542)
<!-- Reviewable:end -->

tbg committed:
roachtest: quiescence with dead node

This test currently fails, at least in remote mode (curiously in local
mode it works, at least on my laptop).

```
--- FAIL: kv/quiescence/nodes=3 [unstable] (1258.49s)
        kv.go:160: QPS dropped by more than 80%: from 2669.32 to 1582.14
```

Fixing this test is the goal for #9446.

Release note: None

tschottdorf commented:
Updated. TFTR!

bors r=petermattis
